core/sql/cli/SessionDefaults.cpp

```diff
@@ -193,6 +194,8 @@ SessionDefaults::SessionDefaults(CollHeap * heap)
   setEspStopIdleTimeout(60);
   // Default is 1800 (idle ESPs time out in 30 minutes)
   setEspIdleTimeout(30*60);
+  // Default is 1800 (Compiler Idle time out in 30 minutes)
+  setCompilerIdleTimeout(30*60);
```

Comment:
Yes. DDL and update statistics would start a new compiler. We could. But, I would leave it for 30 minutes for now to avoid the possibility of increasing the regression run time due to arkcmp startup assuming the regression tests wouldn't take more than 30 minutes between consequent DDL statements.